apps/test/unit/applab/FirebaseStorageTest.js

```diff
@@ -0,0 +1,91 @@
+import { expect } from '../../util/configuredChai';
+import FirebaseStorage from '../../../src/applab/firebaseStorage';
+import { getDatabase, getFirebase } from '../../../src/applab/firebaseUtils';
```

Comment:
Any particular reason you're not using the alias to the src directory? (`@cdo/apps/applab/firebaseStorage` and `@cdo/apps/applab/firebaseUtils`)